app/src/main/kotlin/com/intive/picover/profile/view/ProfileActionsList.kt

```diff
@@ -0,0 +1,60 @@
+package com.intive.picover.profile.view
+
+import android.content.Intent
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationDrawerItem
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import com.intive.picover.R
+import com.intive.picover.profile.model.ProfileActionsListItem
+
+@Composable
+fun ProfileActionsList(onLogoutClick: () -> Unit, onDeleteAccountCLick: () -> Unit) {
```

Comment:
`ProfileActions` would be enough, there is no need decorate the name too much

Reply:
Done